packages/ui/src/bounty/modals/AddBountyModal/components/FundingDetailsStep.tsx

```diff
@@ -0,0 +1,164 @@
+import BN from 'bn.js'
+import React from 'react'
+import styled from 'styled-components'
+
+import { FundingPeriodDetailsContext, GeneralParametersContext } from '@/bounty/modals/AddBountyModal/machine'
+import { InputComponent, InputNumber, Label, ToggleCheckbox } from '@/common/components/forms'
+import { ColumnGapBlock, RowGapBlock } from '@/common/components/page/PageContent'
+import { Tooltip, TooltipContainer, TooltipDefault } from '@/common/components/Tooltip'
+import { TextMedium } from '@/common/components/typography'
+import { BN_ZERO, Colors } from '@/common/constants'
+import { inBlocksDate } from '@/common/model/inBlocksDate'
+import { Address } from '@/common/types'
+
+export interface FundingDetailsStepProps extends Omit<FundingPeriodDetailsContext, keyof GeneralParametersContext> {
+  setFundingMaximalRange: (fundingMaximalRange: BN) => void
+  setFundingMinimalRange: (fundingMinimalRange: BN) => void
+  setCherry: (cherry: BN) => void
+  setFundingPeriodLength: (fundingPeriodLength: BN) => void
+  setFundingPeriodType: (fundingPeriodType: string) => void
+  account?: Address
+}
+
+export const FundingDetailsStep = ({
```

Comment:
Ok, I figured it out. It turned out that I don't have enough transferable JOY amount, but:
1. Calculation is probably wrong because on my account I have 70,935 JOY as transferable, but when I console.log `transferable` it's  only 673 JOY and it's a maximum value that I can fill so that _Next step_ button is enabled 
2. There should be a kind of error message when the cherry is too high